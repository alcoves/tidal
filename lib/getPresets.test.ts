// import { getPresets, clampPreset } from './getPresets'
// import { Metadata } from './getMetadata'

// describe('clampPreset', () => {
//   test('clampPreset generates correct outputs', () => {
//     const options = [
//       [1920, 1080, 1280, 720],
//       [1920, 1080, 1920, 1080],
//       [1920, 1080, 2560, 1440],
//       [1920, 1080, 3840, 2160]
//     ]

//     for (const o of options) {
//       expect(clampPreset(o[0], o[1], o[2], o[3])).toMatchSnapshot()
//     }
//   })
// })

// describe('getPresets', () => {
//   test('that presets are generated', () => {
//     const metadata: Metadata = {
//       width: 1920,
//       height: 1080
//     }
//     expect(getPresets(metadata)).toMatchSnapshot()
//   })
// })
