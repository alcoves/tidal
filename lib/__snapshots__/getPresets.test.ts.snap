// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getPresets that presets are generated 1`] = `
Array [
  Object {
    "defaultMaxRate": 1500,
    "height": 360,
    "name": "360",
    "width": 640,
  },
  Object {
    "defaultMaxRate": 8000,
    "height": 720,
    "name": "720",
    "width": 1280,
  },
  Object {
    "defaultMaxRate": 12000,
    "height": 1080,
    "name": "1080",
    "width": 1920,
  },
]
`;
